### ISSUE TITLE
"Latest" shows as "dev-master"

### ISSUE THREAD

**Issue by chillu** [bringyourownideas/silverstripe-maintenance#126]
It's a bit pointless to show the latest dev branch as a release, right? That's not what we want people to upgrade to. This is on `installer@4.2.0`, not CWP. Maybe it's different in CWP due to recipe constraints?

![image](https://user-images.githubusercontent.com/111025/54639604-b0303080-4af2-11e9-9cab-a6ef424149e5.png)

**Comment by robbieaverill** [comment]
The data we show in this report is essentially just a proxy for `composer outdated`. We haven't built in any layers of customisation yet, but we can definitely do that. If you take `minimum-stability: dev` out of your composer.json you might get different results

**Comment by chillu** [comment]
Good point. Looks like `composer outdated` doesn't have any switches like `--stable-only` (https://getcomposer.org/doc/03-cli.md#outdated). But looking at our installers, both the CWP and core one have `minimum-stability: dev`. So the "latest" column is effectively useless at the moment, we might as well hide it until we create our own logic to determne "latest stable". This is particularly bad since it won't show people on CWP 1.x that CWP 2.x even exists, right? It'll just show `dev-master` everywhere, which doesn't help the decision makers which this module targets. /cc @brynwhyman

**Comment by chillu** [comment]
Maybe that would be an opportunity for a small patch provided back to the composer project, introducing a `composer outdated --minimum-stability` override?
